presto-docs/src/main/sphinx/release/release-0.245.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.245
+=============
+
+**Highlights**
+==============
+* New :func:`enum_key` UDF to get the key for an enum value.
+* Configurable query result compression.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix wrong results issue for queries with multiple lambda expressions that differ only in array/map/row constants that have the same length (:issue:`15424`).
+* Fix serialization bug which causing maps with string-valued enum keys to be displayed as illegible strings in Presto CLI.
+* Fix parsing logic for prepared statements to be consistent with regular statements.
+* Improve memory usage for fragment result caching.
+* Add support for disabling query result compression via client and server-side configuration properties. Clients can disable compressed responses using the ``--disable-compression`` flag or ``disableCompression`` driver property. Compression can be disabled server-wide by using the configuration property: ``query-results.compression-enabled=false``
+* Add support in affinity scheduler to enable cache for bucketed table scan that has remote source.
+* Add :func:`enum_key` to get the key corresponding to an enum value: `ENUM_KEY(EnumType) -> VARCHAR`
```

Comment:
we don't need the signature here. The :func: macro will make this a hyperlink and that should take people to the documentation

Reply:
OK, when I tested it I didn't get a hyperlink.

Reply:
Need  a period at the end of  the line.

Reply:
Going my own way on this and both including it and keeping the description because right now it is the only documentation of it!